### ISSUE TITLE
`expected_event` remove simple argument

### ISSUE THREAD

**Issue by elong0527** [Merck/gsDesign2#264]
Could we remove the `simple` argument?  

I guess user can easily calculate total expected event. Alternatively, we can create another function called `expected_total_event`.

```
# Default arguments, simple output (total event count only)
expected_event(simple = TRUE)
#> [1] 57.3537

# Event count by time period
expected_event(simple = FALSE)
#> # A tibble: 2 × 3
#>       t fail_rate event
#>   <dbl>     <dbl> <dbl>
#> 1     0    0.0770  22.2
#> 2     3    0.0385  35.1
```

**Comment by nanxstats** [comment]
Like what I mentioned in https://github.com/Merck/gsDesign2/issues/262, it would be useful to make functions type stable.

**Comment by LittleBeannie** [comment]
> Could we remove the `simple` argument?
> 
> I guess user can easily calculate total expected event. Alternatively, we can create another function called `expected_total_event`.
> 
> ```
> # Default arguments, simple output (total event count only)
> expected_event(simple = TRUE)
> #> [1] 57.3537
> 
> # Event count by time period
> expected_event(simple = FALSE)
> #> # A tibble: 2 × 3
> #>       t fail_rate event
> #>   <dbl>     <dbl> <dbl>
> #> 1     0    0.0770  22.2
> #> 2     3    0.0385  35.1
> ```

I am okay with the `simple = ...` for now. But I do feel `simple` might not be the correct word, we can call it something like `only_total_event` or something like this. 
I tentatively do not support having `expected_total_event`, since there might be many code duplications.

**Comment by nanxstats** [comment]
I think code duplication is a non-issue because for code reuse you can create an internal heavy-lifting function that will be called by `expected_event()` and `expected_total_event()`.

The main point is still about type stability.

**Comment by LittleBeannie** [comment]
Decision: currently keep `expected_event` as it is considering there might be code duplication (like line 165-183: https://github.com/Merck/gsDesign2/blob/main/R/expected_event.R#L165C3-L184C1)if we created 2 functions.

**Comment by LittleBeannie** [comment]
Let's work on this after #250 is merged.

**Comment by LittleBeannie** [comment]
Decision: Given its low priority and code duplication (referring to `pw_info`'s call of `expected_event`), let's close this issue as not planned for the time being. If it continues to be a persistent problem, we can create a new issue to address it.
